internal/util/dataplane/mode.go

```diff
@@ -1,12 +1,32 @@
 package dataplane
 
+type DBMode string
+
+const (
+	DBModeOff       = "off"
+	DBModePostgres  = "postgres"
+	DBModeCassandra = "cassandra"
```

Comment:
`cassandra` support in Kong Gateway [was removed in 3.4.0](https://docs.konghq.com/gateway/changelog/#breaking-changes-and-deprecations) + KIC now only supports Gateways >= 3.4.1. I think we can drop this one.

Reply:
So we could also remove the checking and special treat of "cassandra" in the code?

Reply:
removed definition of "cassandra" and checking code.